lib/cloud/file-storage/gridfsStorage.js

```diff
@@ -0,0 +1,73 @@
+var q = require('q');
+var fs = require('fs');
+var mongo = require('mongodb');
+var Grid = require('gridfs-stream');
+var MongoClient = require('mongodb').MongoClient
+
+var gfs;
+
+function init(storageConfiguration) {
+  var config = storageConfiguration;
+  if (!config || !config.mongoUrl) {
+    throw Error("Missing mongoUrl parameter for GridFs storage");
+  }
+  MongoClient.connect(config.mongoUrl, function(err, connection) {
```

Comment:
Should this be passing back a `Promise` as `MongoClient.connect` is asynchronous?